frontend/entrypoint.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+echo "window.envVars = {};" > $APP_ROOT/env.js
+
+if [ -n "$VITE_OIDC_AUTHORITY" ]; then
+  echo "window.envVars.VITE_OIDC_AUTHORITY = \"$VITE_OIDC_AUTHORITY\";" >> $APP_ROOT/env.js
```

Comment:
these lines should actually write to `$APP_ROOT/v2/env.js` (the v2 app is in a subfolder called v2)